site/layouts/_default/_markup/render-heading.html

```diff
@@ -0,0 +1,5 @@
+<h{{ .Level }} id="{{ .Anchor | safeURL }}">{{ .Text | safeHTML }}
+{{- if and (ge .Level 2) (le .Level 5) }}{{" " -}}
+<a class="anchor-link" href="#{{ .Anchor | safeURL }}" aria-label="Anchor">#</a>
```

Comment:
`aria-label` value is questionable. @patrickhlauke Do we consider the heading context is enough, or should we try to complete the label using heading's content?

Reply:
yeah i'd probably go for something a bit more descriptive, like having `aria-label="Link to this section: {{ .Text | safeHTML }}"` perhaps. yes, admittedly a bit wordy, but it front-loads what the meaning of the link is and AT users can always choose not to listen to the entire thing

Reply:
Note that I used what anchor.js is already doing. I don't have a preference about this, whatever you guys think is better.